treeherder/perf/management/commands/remove_perf_signatures.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import (BaseCommand,
+                                         CommandError)
+
+from treeherder.perf.models import (PerformanceDatum,
+                                    PerformanceSignature)
+
+
+class Command(BaseCommand):
+
+    help = "Remove performance data"
+    args = '[signature1] [signature2] ...'
+
+    def handle(self, *args, **options):
+        if len(args) < 1:
+            raise CommandError("Need to specify at least one signature to "
+                               "remove")
+
+        for signature_hash in args:
+            signatures = PerformanceSignature.objects.filter(
+                signature_hash=signature_hash)
+            datums = PerformanceDatum.objects.filter(
```

Comment:
If there are no signatures matching the hash, this script will throw an exception. I think that's the right behavior.